src/FirebaseHttpClient_Esp8266.cpp

```diff
@@ -15,12 +15,44 @@
 #define USE_ESP_ARDUINO_CORE_2_0_0
 #endif
 
+// Firebase now returns `Connection: close` after REST streaming redirection.
+//
+// Override the built-in ESP8266HTTPClient to *not* close the
+// connection if forceReuse it set to `true`.
+class ForceReuseHTTPClient : public HTTPClient {
+public:
+  void end() {
+    if(connected()) {
```

Comment:
I like that better only because it deviates less from the base so as the base changes we can merge with impunity.